freqtrade/templates/FreqaiExampleStrategy.py

```diff
@@ -161,6 +173,10 @@ def set_freqai_targets(self, dataframe, **kwargs):
         Required function to set the targets for the model.
         All targets must be prepended with `&` to be recognized by the FreqAI internals.
 
+        Access metadata such as the current pair with:
```

Comment:
I guess the "metadata" kwarg should be in the header definition for the functions (in the sample strategies, as well as documentation).

Right now we say "access via metadata" - but unless you implicitly know that metadata is a 2nd argument - you'll just assume it's "magically" there - which is not the case.

Reply:
Ok, I have added metadata to the docstrings as well as the example arguments.